pkg/installers/windows/Kovri32.iss

```diff
@@ -0,0 +1,90 @@
+; Kovri Installer for Windows, 32 bit variant
+; Copyright (c) 2013-2017, The Kovri I2P Router Project

```

Comment:
Yes, of course :) I just copied this from some other files without really checking...